### PR TITLE
RTCMediaStreamTrackStats: framesCaptured added.

### LINKED ISSUE #159: Stat for camera input framerate of local video tracks
RTCMediaStreamTrackStats.framesPerSecond, frames[Sent/Received] and RTC[In/Out]boundRTPStreamStats.frames[Encoded/Decoded] are available to find out the configures and actual (degraded) framerates.

However, there is no stat for the actual camera input framerate. This can heavily vary due to lighting conditions, and it's useful to distinguish between degradation due to device/environment and due to the rest of the pipeline/network.

Should we add `unsigned long framesCaptured` to local (`isRemote=false`) `RTCMediaStreamTrackStats`?

### PR DESCRIPTION
**Pull request by henbos**
Fixes #159.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -1300,6 +1301,19 @@
...
+                <dfn><code>framesCaptured</code></dfn> of type <span class="idlMemberType"><a>
+                unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for local video. It represents the total number of frames captured for
+                  this MediaStreamTrack. For example, if this track represents a camera this is the
+                  number of frames produced by the camera for this track, whose framerate could
+                  vary due to hardware limitations or environmental factors such as lighting
+                  condition.
```
Comment (vr000m) [inline webrtc-stats.html]: conditions

File: webrtc-stats.html [hunk 2]
```diff
@@ -1300,6 +1301,19 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>framesCaptured</code></dfn> of type <span class="idlMemberType"><a>
+                unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for local video. It represents the total number of frames captured for
+                  this MediaStreamTrack. For example, if this track represents a camera this is the
```
Comment (vr000m) [inline webrtc-stats.html]: Should we clarify that this is before or after encoding is done.

File: webrtc-stats.html (hunk 1 above)
Comment (henbos) [inline webrtc-stats.html]: Done.

File: webrtc-stats.html (hunk 2 above)
Comment (henbos) [inline webrtc-stats.html]: Done.

**Pull request merged**